<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BalanceForm</class>
 <widget class="QWidget" name="BalanceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>529</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>529</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>529</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Myanmar Text</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Balance</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>529</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="background.qrc">:/image/Better Template Forms.jpg</pixmap>
   </property>
  </widget>
  <widget class="QComboBox" name="accountComboBox">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>240</y>
     <width>290</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(4, 27, 48);
color: rgb(255, 255, 255);
font: 12pt &quot;Hyogo&quot;;
border-radius : 8px;
border : 1px solid #ffffff;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="balanceLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>18</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font:700 33pt &quot;Hyogo&quot;;</string>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="QLabel" name="accountLabel">
   <property name="geometry">
    <rect>
     <x>167</x>
     <y>221</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(0, 0, 0);
font:700 18pt &quot;Hyogo&quot;;
</string>
   </property>
   <property name="text">
    <string>Account : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="showBalancePushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>317</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(4, 27, 48);
color: rgb(255, 255, 255);
font: 12pt &quot;Hyogo&quot;;
border-radius : 8px;
border : 1px solid #ffffff;
</string>
   </property>
   <property name="text">
    <string>Show Balance</string>
   </property>
  </widget>
  <widget class="QLabel" name="footerBalanceLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>356</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(0, 0, 0);
font:700 18pt &quot;Hyogo&quot;;</string>
   </property>
   <property name="text">
    <string>Balance : </string>
   </property>
  </widget>
  <widget class="QLabel" name="valueLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>376</y>
     <width>280</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(0, 0, 0);
font:18pt &quot;Hyogo&quot;;</string>
   </property>
   <property name="text">
    <string>value</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backPushButton">
   <property name="geometry">
    <rect>
     <x>609</x>
     <y>125</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(4, 27, 48);
color: rgb(255, 255, 255);
font: 12pt &quot;Hyogo&quot;;
border-radius : 8px;
border : 1px solid #ffffff;
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="nationalCodeLabel">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>126</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 11pt &quot;Hyogo&quot;;</string>
   </property>
   <property name="text">
    <string>National Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="accountComboBoxErrorLabel">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>274</y>
     <width>284</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Hyogo&quot;;
color: rgb(255, 73, 73);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="firstNameLabel">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>85</y>
     <width>261</width>
     <height>49</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 14pt &quot;Hyogo&quot;;</string>
   </property>
   <property name="text">
    <string>First Name And LastName</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
